docs/Sprint-Review/sprint-94-summary.md

```diff
@@ -0,0 +1,80 @@
+# Sprint 94 Summary
+
+02/28/2024 - 03/12/2024
+
+Velocity (Dev): 16
```

Comment:
@reitermb please double-check velocity with changes mentioned below.

Reply:
Velocity should still be right given that #2825 merged in today rather than yesterday!